chainer/training/extensions/graph_report.py

```diff
@@ -0,0 +1,141 @@
+from chainer import reporter
```

Comment:
Could you move this line to the bottom of the import block? Because Import statements must be organized into three parts in this order: standard libraries, third-party libraries, and internal imports. I'd be happy if you would check this contribution guide here :) http://docs.chainer.org/en/stable/contribution.html